onions.md

```diff
@@ -2,26 +2,26 @@
 
 > This is a retooling of the [raw version](onions-raw.md).  See that document for reference.
 
-The Operationalizing Network & service abstractIONS (ONIONS) Working Group aims to make it easier to implement and use the IETF-wide service and network abstractions work, with the goal of improving network automation, operational efficiency, and interoperability.
+The Operationalizing Network & service abstractIONS (ONIONS) Working Group aims to make it easier to implement and use the IETF's service and network abstractions, with the goal of improving network automation, operational efficiency, and interoperability.
 
-Abstraction, for the purposes of this working group, refers to the process of defining simplified, high-level constructs that represent network and service-level capabilities, enabling interaction and automation without directly exposing the underlying device-specific implementations.
+For the purposes of this working group, the term "Abstraction" refers to the process of defining simplified, high-level constructs that represent network and service-level capabilities. Abstraction enables interaction between management components, and automation of management systems without directly exposing the underlying device-specific implementations.
 
-To achieve this, ONIONS will serve as the focal point for network and service abstraction modeling in the IETF. This includes (1) providing a comprehensive overview of operational needs and motivations for network and service abstractions, (2) structuring efforts on service and network models, including their maintenance and integration with underlying device models, (3) defining and maintaining models for network and service abstractions (e.g., L3SM, L2SM, L2NM, L3NM, ACaaS, SAP, Slicing, SAIN, VN, and ACTN) based on operational feedback, and (4) exploring approaches for mapping between various abstraction layers to enable seamless interoperability.
+To achieve this, ONIONS will serve as the focal point for network and service abstraction modeling in the IETF. This includes (1) providing a comprehensive overview of operational needs and motivations for network and service abstractions, (2) structuring efforts on service and network models, including their maintenance and exploring their relationship with underlying device models, (3) defining and maintaining models for network and service abstractions (e.g., L3SM, L2SM, L2NM, L3NM, ACaaS, SAP, Slicing, SAIN, VN, ACTN, and the automating service and network management framework) based on operational feedback, and (4) exploring approaches for mapping between various abstraction layers to enable seamless interoperability.
```

Comment:
```suggestion
To achieve this, ONIONS will serve as the focal point for network and service abstraction modeling in the IETF. This includes (1) providing a comprehensive overview of operational needs and motivations for network and service abstractions, (2) structuring efforts on service and network models, including their maintenance and how they interact with underlying device models, (3) defining and maintaining models for network and service abstractions (e.g., L3SM, L2SM, L2NM, L3NM, ACaaS, SAP, Slicing, SAIN, VN, ACTN, and the automating service and network management framework) based on operational feedback, and (4) exploring approaches for mapping between various abstraction layers to enable seamless interoperability.
```

Reply:
avoid "exploring" as this smells more "researchy" for some. Just save us a comment.

Reply:
We could consider breaking this into a numbered list to improve readability, rather than keeping it as a single large paragraph.